pkgs/development/embedded/jtag-remote-server/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "jtag-remote-server";
-  version = "unstable-2022-06-09";
+  version = "unstable-2022-10-28";
 
   src = fetchFromGitHub {
     owner = "jiegec";
     repo = pname;
-    rev = "917d8d298423ba1aa6e75aa92e009b7f27f74a57";
-    hash = "sha256-Jy0OyRgn9SYpjP3HYWPvRirfxXk4/vMYvZuI3XpPtBw=";
+    rev = "e12e5b20838d329aea68de7a8028051e0579c003";
```

Comment:
Let's see if we can get [tags](https://github.com/jiegec/jtag-remote-server/issues/1).